test/integration/provisionedConcurrency.test.js

```diff
@@ -13,24 +14,29 @@ describe('AWS - Provisioned Concurrency Integration Test', function() {
   let stackName;
   let servicePath;
   let queueName;
+  let streamName;
   const stage = 'dev';
 
   before(async () => {
     const serviceData = await fixtures.setup('provisionedConcurrency');
     ({ servicePath } = serviceData);
     const serviceName = serviceData.serviceConfig.service;
 
+    streamName = `${serviceName}-kinesis`;
     queueName = `${serviceName}-provisioned`;
     stackName = `${serviceName}-${stage}`;
-    // create existing SQS queue
-    // NOTE: deployment can only be done once the SQS queue is created
+    // NOTE: deployment can only be done once the SQS queue and Kinesis Stream is created
     log.notice(`Creating SQS queue "${queueName}"...`);
     await createSqsQueue(queueName);
+    log.notice(`Creating Kinesis stream "${streamName}"...`);
+    await createKinesisStream(streamName);
```

Comment:
I think we can create them in parallel (?)

Reply:
of course - great idea :+1: